followthemoney/schema/License.yaml

```diff
@@ -19,6 +19,12 @@ License:
   properties:
     area:
       label: "Area"
+      type: number
+      description: "Spatial area of the license or concession"
+    location:
+      label: "Location"
+      type: address
+      description: "Location of the license or concession"
```

Comment:
Is this necessary? `License` already inherits `address` and `addressEntity` from `Thing`.

Reply:
You're totally right. Let's kick it out.